src/Compilers/Core/AnalyzerDriver/DeclarationComputer.cs

```diff
@@ -13,9 +13,7 @@ internal class DeclarationComputer
         internal static DeclarationInfo GetDeclarationInfo(SemanticModel model, SyntaxNode node, bool getSymbol, IEnumerable<SyntaxNode> executableCodeBlocks, CancellationToken cancellationToken)
         {
             var declaredSymbol = getSymbol ? model.GetDeclaredSymbol(node, cancellationToken) : null;
-            var codeBlocks = executableCodeBlocks == null ?
-                ImmutableArray<SyntaxNode>.Empty :
-                executableCodeBlocks.Where(c => c != null).AsImmutableOrEmpty();
+            var codeBlocks = executableCodeBlocks?.Where(c => c != null).AsImmutableOrEmpty() ?? ImmutableArray<SyntaxNode>.Empty;
```

Comment:
Here is why I like `?.` form more: it avoids a duplication of `executableCodeBlocks` name, so you do not have to visually compare them to make sure it's the same variable (e.g. maybe one has a `_` prefix and is a field?). In case it operates on a field or captured local, you know that it's atomic and you do not need to think of a possibility of a race condition.

But if most people do not see any value in it, I am ready to revert such changes.